spec/count_words_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rspec'
+require 'count_words'
+
+describe 'initializing counter' do
+  let(:counter) { CountWords.new }
+
+  expected = {}
```

Comment:
This being outside the `it` block is strange to me. I don't think the local variable adds much value here. `.to eq({})`